### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/extra-models.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/extra-models.md to Japanese.
Please review tutorial/extra-models.md

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.
I will fix dependency after #1571 & #1656 are merged.~

### CODE REVIEW

File: docs/ja/docs/tutorial/extra-models.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+```Python
+UserInDB(
+    username = user_dict["username"],
+    password = user_dict["password"],
+    email = user_dict["email"],
+    full_name = user_dict["full_name"],
+)
+```
+
+#### 別の内容からのPydanticモデル
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/extra-models.md:95]: #### 別のモデルからつくるPydanticモデル

File: docs/ja/docs/tutorial/extra-models.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+このようにして、モデル間の違いだけを宣言することができます:
+
+```Python hl_lines="9  15 16  19 20  23 24"
+{!../../../docs_src/extra_models/tutorial002.py!}
+```
+
+## `Union`または`anyOf`
+
+レスポンスを２つの型の`Union`と宣言することができます。
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/extra-models.md:159]: レスポンスを２つの型の`Union`として宣言することができます。

**Review by tokusumi — CHANGES_REQUESTED** [review]

File: docs/ja/docs/tutorial/extra-models.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+```Python
+UserInDB(**user_in.dict(), hashed_password=hashed_password)
+```
+
+...以下のようになります:
+
+```Python
+UserInDB(
+    username = user_dict["username"],
+    password = user_dict["password"],これやばい
```
[the remark this answers is not in the document]
Comment (SwftAlpc) [reply docs/ja/docs/tutorial/extra-models.md]: 変な日本語入ってましたね...;;
修正しました！

File: docs/ja/docs/tutorial/extra-models.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
+# 追加モデル
```
[the remark this answers is not in the document]
Comment (SwftAlpc) [reply docs/ja/docs/tutorial/extra-models.md]: 良い和訳だと思います！修正しました

**Comment by SwftAlpc** [comment]
@tokusumi 
Thanks for the review!
I apologize for taking so long to respond.
Could you re-review?

**Review by tokusumi — APPROVED** [review]
your awesome works are really helpful. Thank you🎉

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4b6e845ef8e2993f1d1f1ef0a745170591e132b4 at: https://6499c2f1bac8cd0083555671--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this was done by @SwftAlpc and reviewed by @tokusumi, champions of the Japanese translations, I'm gonna merge this. 🚀

**Pull request merged**